packages/compass-schema-validation/src/components/validation-editor.spec.tsx

```diff
@@ -245,4 +245,38 @@ describe('ValidationEditor [Component]', function () {
       expect(applyBtn).to.have.attribute('aria-disabled', 'true');
     });
   });
+
+  context('with errorAndLog', function () {
+    it('should be hidden for server version <8.1', async function () {
+      await renderValidationEditor({
+        serverVersion: '6.0.0',
+      });
+
+      expect(
+        screen.queryByTestId('validation-action-error-and-log-option')
+      ).is.null;
+
+      screen.getByTestId('validation-action-selector').click();
```

Comment:
Small nit, not sure if there's some special reason to do it that way, but testing library recomments using userEvent for this (you can see it being done in other tests in this file)

```suggestion
      userEvent.click(screen.getByTestId('validation-action-selector'))
```